interface/fax/fax_view.php

```diff
@@ -35,41 +39,84 @@
     }
 
     while (!feof($jfhandle)) {
-        $line = trim(fgets($jfhandle));
-        if (substr($line, 0, 12) == '!postscript:') {
-            $ffname = $GLOBALS['hylafax_basedir'] . '/' .
-                substr($line, strrpos($line, ':') + 1);
-            break;
-        }
-    }
+	    $line = trim(fgets($jfhandle));
+
+	    if (strstr($line, '!')) {
+		    $docs[] = $GLOBALS['hylafax_basedir'] . '/' . substr($line, strrpos($line, ":")+1);
+	    }
 
+    }
     fclose($jfhandle);
-    if (!$ffname) {
-        die(xlt("Cannot find postscript document reference in ") . text($jfname));
+
+    $tmp1 = array();
+    $tmp2 = 0;
+    $inames = '';
+    $faxcache = $GLOBALS['OE_SITE_DIR'] . "/faxcache/tmp";
+    if (!is_dir($faxcache)) {
+	    exec("mkdir -p " . escapeshellarg($faxcache));
+    }
+    foreach ($docs as $filepath) {
+	    $filebase = basename($filepath) . ".pdf";
+	    if (preg_match('/ps/', $filepath)) {
+		    $tmp0 = exec("ps2pdf " . escapeshellarg($filepath) . " " . escapeshellarg($faxcache) . "/" . escapeshellarg($filebase), $tmp1, $tmp2);
+		    if ($tmp2) {
+			    die(xlt("ps2pdf returned $tmp2: $tmp0 "));
```

Comment:
would use text() rather than xlt her and in below die (no reason to translate and if do translate the variable should not be included in the constants)

Reply:
I think I understand now.  Will change.